app/assets/javascripts/components/Columns/ColumnItem.js

```diff
@@ -1,28 +1,34 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Column = ({ title, children, renderAction, onClose }) => (
-  <div className="Column">
-    <div className="Column__header">
-      <h3 className="Column__name">{title}</h3>
-      <div className="Column__actions">
-        {renderAction()}
-        <button type="button" className="Column__btn-close" onClick={onClose}>
-          <i className="mi md-light md-16">close</i>
-        </button>
+const Column = ({ title, children, renderAction, onClose, visible }) => (
+  visible &&
+    <div className="Column" data-id="column">
```

Comment:
Can we wrap it in parenthesis to make it clearer?

```suggestion
  visible && (
    <div className="Column" data-id="column">
    // ....
  )
```

Reply:
Sure!